iot-e2e-tests/common/src/test/resources/log4j.properties

```diff
@@ -5,4 +5,10 @@ log4j.rootLogger=ERROR, stdout
 log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.Target=System.out
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
-log4j.logger.tests.integration.com.microsoft.azure.sdk.* = DEBUG 
\ No newline at end of file
+log4j.logger.tests.integration.com.microsoft.azure.sdk.* = DEBUG
```

Comment:
We don't really do different log levels in our e2e test packages, so I'm leaving this as DEBUG